ui/selectors/multichain/networks.ts

```diff
@@ -113,16 +108,16 @@ export const getMultichainNetworkConfigurationsByChainId =
 
       // This is not ideal but since there are only two non EVM networks
       // we can just filter them out based on the support enabled
-      const { bitcoinEnabled, solanaEnabled } = isNonEvmNetworksEnabled;
+      const { bitcoinEnabled } = isNonEvmNetworksEnabled;
       if (bitcoinEnabled) {
         filteredNonEvmNetworkConfigurationsByChainId[BtcScope.Mainnet] =
           nonEvmNetworkConfigurationsByChainId[BtcScope.Mainnet];
       }
 
-      if (solanaEnabled) {
-        filteredNonEvmNetworkConfigurationsByChainId[SolScope.Mainnet] =
-          nonEvmNetworkConfigurationsByChainId[SolScope.Mainnet];
-      }
+      ///: BEGIN:ONLY_INCLUDE_IF(solana)
+      filteredNonEvmNetworkConfigurationsByChainId[SolScope.Mainnet] =
+        nonEvmNetworkConfigurationsByChainId[SolScope.Mainnet];
+      ///: END:ONLY_INCLUDE_IF
```

Comment:
Since `solanaEnabled` is already code-fenced, I would just keep using the runtime `if` for this? Also, the `solanaEnabled = true;` should probably be using the remote-ff-flag rather than defaulting to `true`, no?
```suggestion
      if (solanaEnabled) {
        filteredNonEvmNetworkConfigurationsByChainId[SolScope.Mainnet] =
          nonEvmNetworkConfigurationsByChainId[SolScope.Mainnet];
      }
```